src/python/pants/core/target_types.py

```diff
@@ -76,7 +76,7 @@
 
 @dataclass(frozen=True)
 class per_platform(Generic[_T]):
-    """An object containing differing homogeneous platform-dependent values.
+    """An object containing differing homogenous platform-dependent values.
```

Comment:
I think "homogeneous" is actually more correct. If I understand correctly, "homogenous" is archaic and not commonly used.

Reply:
yeah. weird that the spellchecker suggested this. reverting this one...